cisco_aci/changelog.d/18360.fixed

```diff
@@ -0,0 +1 @@
+[NDM] [Cisco ACI] Refactor batched payloads to fix incorrect status
```

Comment:
Is this changelog needed?  This is fixing a new bug that is not in an agent release yet, right?

Reply:
The changelog is needed since it's already released on the release branch!

Reply:
amended to incl. the changelog again!